test/test-remote/test_loki_log_ingest_api.ts

```diff
@@ -322,8 +322,8 @@ suite("Loki API test suite", function () {
             uniq: rndstring().slice(0, 5)
           },
           values: [
-            ["1286705401123456789", "aaa"],
-            ["1286705401123456789", "aaa"]
+            [now, "aaa"],
+            [now, "aaa"]
```

Comment:
update: see discussion above. This change makes sense because we do not want to test inserting data older than the retention period.